src/views/virtualmachines/list/components/Timestamp/datetime.ts

```diff
@@ -0,0 +1,113 @@
+const maxClockSkewMS = -60000;
```

Comment:
This seems like something that could be used in multiple places throughout the code. @yaacov If timestamps like this need to be used elsewhere, would this be moved to a common location or is this the type of thing that you want recreated individually per view?